packages/story-editor/src/components/mediaRecording/useTrim.js

```diff
@@ -17,30 +17,46 @@
 /**
  * External dependencies
  */
+import { formatMsToHMS } from '@googleforcreators/media';
 import { useState, useCallback } from '@googleforcreators/react';
 
-function useTrim(setDuration) {
+/**
+ * Internal dependencies
+ */
+import useFFmpeg from '../../app/media/utils/useFFmpeg';
+
+function useTrim({ setDuration, onTrimmed, file }) {
   const [trimData, setTrimData] = useState({ start: 0, end: null });
   const [isTrimming, setIsTrimming] = useState(false);
+  const [isProcessingTrim, setIsProcessingTrim] = useState(false);
+  const { trimVideo } = useFFmpeg();
   const onTrim = useCallback(
-    (newTrimData) => {
+    async (newTrimData) => {
+      setIsTrimming(false);
```

Comment:
Hm, should this be set to `false` after the trimming has actually happened, after `newTrimData` as it was before? It seems that it's set to false right away `onTrim` currently?